coalib/results/Diff.py

```diff
@@ -221,7 +221,7 @@ def affected_code(self, filename):
         return list(diff.range(filename)
```

Comment:
@jarifibrahim we usually refrain from using words like `Fix` in the commit shortlog - they talk more about the issue than the commit itself. Maybe use something like `Diff: Change default distance to -1`? (You can drop the `.py` part in the shortlog.)